backend/battling/views.py

```diff
@@ -24,16 +20,18 @@ class CreateBattle(CreateView):
 
     def form_valid(self, form):
         form.instance.creator = self.request.user
+        form.instance.status = "ONGOING"
```

Comment:
When using [enumeration choices](https://docs.djangoproject.com/en/3.2/ref/models/fields/#enumeration-types), you could do something like `Battle.Status.ONGOING`. That prevents typos and using non-existing choices.

Reply:
I tried to use it, but I can't get it to work

Reply:
What error do you get?

Reply:
did you follow the same architecture described here? https://docs.djangoproject.com/en/3.1/ref/models/fields/#enumeration-types

if so, please paste your model here or send me via Slack so I can evaluate

Reply:
Ok, I'll send you via Slack then :D